DataFormats/L1TrackTrigger/interface/TTTrack.h

```diff
@@ -115,268 +136,232 @@ class TTTrack {
 template <typename T>
 TTTrack<T>::TTTrack() {
   theStubRefs.clear();
-  theMomentum4Par = GlobalVector(0.0, 0.0, 0.0);
-  theMomentum5Par = GlobalVector(0.0, 0.0, 0.0);
-  theRInv4Par = 0.0;
-  theRInv5Par = 0.0;
-  thePOCA4Par = GlobalPoint(0.0, 0.0, 0.0);
-  thePOCA5Par = GlobalPoint(0.0, 0.0, 0.0);
-  theSector = 0;
-  theWedge = 0;
-  theChi24Par = 0.0;
-  theChi25Par = 0.0;
-  theStubPtConsistency4Par = 0.0;
-  theStubPtConsistency5Par = 0.0;
-  valid4ParFit = false;
-  valid5ParFit = false;
+  theMomentum = GlobalVector(0.0, 0.0, 0.0);
+  theRInv = 0.0;
+  thePOCA = GlobalPoint(0.0, 0.0, 0.0);
+  thePhiSector = 0;
+  theTrackSeed = 0;
+  theChi2 = 0.0;
+  theStubPtConsistency = 0.0;
+  NumFitPars = 0;
 }
 
 /// Another Constructor
 template <typename T>
 TTTrack<T>::TTTrack(std::vector<edm::Ref<edmNew::DetSetVector<TTStub<T> >, TTStub<T> > > aStubs) {
   theStubRefs = aStubs;
-  theMomentum4Par = GlobalVector(0.0, 0.0, 0.0);
-  theMomentum5Par = GlobalVector(0.0, 0.0, 0.0);
-  theRInv4Par = 0.0;
-  theRInv5Par = 0.0;
-  thePOCA4Par = GlobalPoint(0.0, 0.0, 0.0);
-  thePOCA5Par = GlobalPoint(0.0, 0.0, 0.0);
-  theSector = 0;
-  theWedge = 0;
-  theChi24Par = 0.0;
-  theChi25Par = 0.0;
-  theStubPtConsistency4Par = 0.0;
-  theStubPtConsistency5Par = 0.0;
-  valid4ParFit = false;
-  valid5ParFit = false;
+  theMomentum = GlobalVector(0.0, 0.0, 0.0);
+  theRInv = 0.0;
+  thePOCA = GlobalPoint(0.0, 0.0, 0.0);
+  thePhiSector = 0;
+  theTrackSeed = 0;
+  theChi2 = 0.0;
+  theStubPtConsistency = 0.0;
+  NumFitPars = 0;
+}
+
+/// Meant to be default constructor
+template <typename T>
+TTTrack<T>::TTTrack(double aRinv,
+                    double aphi,
+                    double aeta,
+                    double az0,
+                    double ad0,
+                    double aChi2,
+                    unsigned int aHitPattern,
+                    unsigned int nPar,
+                    double aBfield) {
+  theStubRefs.clear();
+  double thePT = 0.3 * aRinv * aBfield;
```

Comment:
magic number 0.3 should be a named constant

Reply:
fixed